.travis.yml

```diff
@@ -8,8 +8,7 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
+  - 7.2
```

Comment:
Consider adding the current PHP version (7.4) to see if we start introducing incompatibilities with that version during development.

Reply:
Yes, but lets do that in a different PR.